src/components/Footer.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import { FaArrowRight} from 'react-icons/fa';
+import { Link } from 'react-router-dom';
+import { GrMailOption } from 'react-icons/gr';
+import { IconContext } from "react-icons";
+
+import {
+  AiFillLinkedin,
+  AiFillTwitterSquare,
+  AiFillFacebook,
+  AiOutlineWhatsApp
+} from 'react-icons/ai';
+import { nameLogo } from '../assets';
+
+const Footer = () => {
+  const [email, setEmail] = useState('');
+  return (
+    <div className='footer'>
+      {/* footer logo ______________  */}
+      <div className='footer-logo'>
+        <img src={nameLogo} alt='logo' />
+      </div>
+
+      <div className='footer-contact'>
+        <h2>Office</h2>
+        <ul>
+          <li>KG 11 Av</li>
+          <li>Kigali, Gasabo, Imena street</li>
+          <li>techsmith@gmail.com</li>
+          <li>+250 788 000 000</li>
+        </ul>
+      </div>
+
+      <div className='footer-links'>
+        <h2>Links</h2>
+        <ul>
+          <li>
+            <Link style={{ textDecoration: "none", color: '#fff' }} to='/'> Home </Link>
+          </li>
+          <li>
+
+            <Link style={{ textDecoration: "none", color: '#fff' }} to='/'> About </Link>
+          </li>
+          <li>
+
+            <Link style={{ textDecoration: "none", color: '#fff' }} to='/'> Collection </Link>
+          </li>
+          <li>
+
+            <Link style={{ textDecoration: "none", color: '#fff' }} to='/'> Products </Link>
+          </li>
+          <li>
+
+            <Link style={{ textDecoration: "none", color: '#fff' }} to='/'> Chat </Link>
+          </li>
```

Comment:
The same applies here, instead of having a lot of lines of code, you can use map to map over all the list items, and have the items in constants/index.js file